src/general/OptionsMenu.cs

```diff
@@ -769,6 +775,54 @@ private Viewport.MSAA MSAAIndexToResolution(int index)
         }
     }
 
+    private int MaxFPSValueToIndex(int value)
+    {
+        switch (value)
+        {
+            case 30:
+                return 0;
+            case 60:
+                return 1;
+            case 90:
+                return 2;
+            case 120:
+                return 3;
+            case 144:
+                return 4;
+            case 240:
+                return 5;
+            case 0:
+                return 6;
+            default:
```

Comment:
An exception should be thrown in invalid value

Reply:
Instead of an explicit exception, would printing an error and returning the default value also work (similar to the `MSAAIndexToResolution()` function above it)?

Reply:
If that's what's done elsewhere in the options, then it'll be good to be consistent with existing code.